python/tchannel/messages/base.py

```diff
@@ -25,3 +25,15 @@ def __str__(self):
             str(self.__class__.__name__),
             ", ".join(attrs)
         )
+
+    def encode(self):
+        """ Encode all the strings in the msg using
+        encode type from common.py
```

Comment:
What does this mean?

Also, for placeholders like this, use `raise NotImplementedError()` instead of `pass`.